third_party/terraform/website/docs/d/sql_database_instance.html.markdown

```diff
@@ -16,7 +16,7 @@ Use this data source to get information about a Cloud SQL instance
 
 ```hcl
 data "google_sql_database_instance" "qa" {
-    name = google_sql_database_instance.master.name
+    name = "test-sql-instance"
```

Comment:
I believe the intention was to imply that you could use the same name from your resource to your datasource. But I can see how it may be unclear, so this should be good